CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ _None_
 
 ### Internal Changes
 
-_None_
+- Updates `activesupport` to `6.1.7.1`, addressing [a security issue](https://github.com/advisories/GHSA-j6gc-792m-qgm2). This is a major version change, but as the dependency is internal-only, it shouldn't be a breaking change for clients. [#441]
```

Comment:
> as the dependency is internal-only, it shouldn't be a breaking change for clients.

I don't think we'll run into this, but what if another library in the client's `Gemfile` has a version rule that is incompatible with this one? Would that count as a breaking change? 🤔 

Also, I wanted to read more about `add_gem` and the concept of "internal dependency". In the context of the sentence above, I read that as "hidden from the clients," but that's not my understanding as [they do appear in the `Gemfile.lock` ](https://github.com/wordpress-mobile/WordPress-iOS/blob/0e45ee02ba955ba45ec8db37d5d2e1faa1b9190a/Gemfile.lock#L6-L10). 

Anyways, I was surprised to discover that `add_dependency` is no longer an option. The [documentation](https://guides.rubygems.org/specification-reference/) only shows `add_development_dependency` and `add_runtime_dependency`.

I haven't investigated it further but my guess is that it's either deprecated and no longer advertised, or that the removal happened in a later tooling version than ours, which is why we haven't had issues so far.

Reply:
FWIW, this works on WordPress iOS. See https://github.com/wordpress-mobile/WordPress-iOS/pull/20012.